lxd-migrate/main_migrate.go

```diff
@@ -5,37 +5,34 @@ import (
 	"io/ioutil"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"runtime"
 	"sort"
 	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"golang.org/x/sys/unix"
+	"gopkg.in/yaml.v2"
 
+	lxd "github.com/lxc/lxd/client"
```

Comment:
Incorrect import name

Reply:
Well, correct name but shouldn't have the name next to it as it's its default name.